core/src/main/scala/scalaz/MonadError.scala

```diff
@@ -11,6 +11,9 @@ trait MonadError[F[_], S] extends Monad[F] { self =>
   def raiseError[A](e: S): F[A]
   def handleError[A](fa: F[A])(f: S => F[A]): F[A]
 
+  def emap[A, B](fa: F[A])(f: A => S \/ B): F[B] =
+    bind(fa)(a => f(a).fold(raiseError(_), pure(_)))
+
```

Comment:
override with `emap = bind` for `\/`?

Reply:
oh, of course, I forgot that. Same for Either.